recipe/build-pip-package.sh

```diff
@@ -61,8 +61,25 @@ echo "RECIPE_DIR: $RECIPE_DIR"
 
 # Cache libtensorflow libs in SRC_DIR so they can be picked up by
 # the libtensorflow output
+TF_MAJOR_VERSION=${PKG_VERSION:0:1}
+echo "TF_MAJOR_VERSION: $TF_MAJOR_VERSION"
+
 mv ${SP_DIR}/tensorflow/libtensorflow.so ${SRC_DIR}/tensorflow_pkg/
 mv ${SP_DIR}/tensorflow/libtensorflow_cc.so ${SRC_DIR}/tensorflow_pkg/
+mv ${SP_DIR}/tensorflow/libtensorflow_framework.so.${TF_MAJOR_VERSION} ${SRC_DIR}/tensorflow_pkg/
+
+#Cache the libs and headers needed by libtensorflow
+
+CACHE_DIR="/tmp/tf-libs"
```

Comment:
I tried to put the value of CACHED_DIR into config/conda_build_config.yaml so that it need not be hardcoded in both build scripts, but somehow I can't access that variable in build scripts, if it is defined in conda_build_config.yaml.

Reply:
yeah, it's a bit unfortunate that this has to be hardcoded in the feedstock, but it should be ok.